### PR TITLE
signup-signin

### PR DESCRIPTION
**Pull request by ngocvnn**

### CODE REVIEW

File: app/views/devise/confirmations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div class="row">
+  <div class="col-md-4 col-md-offset-4">
+    <h2>Resend confirmation instructions</h2>
+
+    <%= form_for resource, as: resource_name, 
+      url: confirmation_path(resource_name), html: { method: :post } do |f| %>
```
Comment (PhamVanTung) [inline app/views/devise/confirmations/new.html.erb]: indent

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+
   root 'static_pages#home'
   get 'help' => 'static_pages#help'
   get 'about' => 'static_pages#about'
   get 'contact' => 'static_pages#contact'
+
+  devise_for :users
+
```
Comment (nguyenngoctruong) [inline config/routes.rb]: remove empty line

File: app/views/devise/passwords/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<div class="row">
+  <div class="col-md-4 col-md-offset-4">
+    <h2>Change your password</h2>
+
+    <%= form_for resource, as: resource_name, 
+      url: password_path(resource_name), html: { method: :put } do |f| %>
```
Comment (PhamVanTung) [inline app/views/devise/passwords/edit.html.erb]: indent

File: app/views/devise/passwords/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<div class="row">
+  <div class="col-md-4 col-md-offset-4">
+    <h2>Forgot your password?</h2>
+
+    <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
```
Comment (PhamVanTung) [inline app/views/devise/passwords/new.html.erb]: indent

File: app/views/devise/unlocks/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend unlock instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f| %>
```
Comment (PhamVanTung) [inline app/views/devise/unlocks/new.html.erb]: indent

File: app/views/devise/registrations/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+        <div class="field">
+          <%= f.label :email %><br />
+          <%= f.email_field :email, autofocus: true %>
+        </div>
+
+        <div class="field">
+          <%= f.label :password %>
+          <% if @validatable %>
+          <em>(<%= @minimum_password_length %> characters minimum)</em>
```
Comment (dieunb) [inline app/views/devise/registrations/new.html.erb]: indent

File: config/environments/development.rb [hunk 7]
```diff
@@ -38,4 +38,5 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.action_mailer.default_url_options = {:host => 'localhost:3000'}
```
Comment (dieunb) [inline config/environments/development.rb]: coding convention

**Pull request merged**